### PR TITLE
Post Hybrid

### PR DESCRIPTION
**Pull request by Justin-Vuong**

### CODE REVIEW

File: vcu/Src/drive_by_wire.c [hunk 1]
```diff
@@ -118,17 +119,17 @@ uint32_t EM_Enable(uint32_t event)
...
 
     if (!bpsState) {
         DEBUG_PRINT("Failed to em enable, bps fault\n");
         sendDTC_WARNING_EM_ENABLE_FAILED(0);
         state = STATE_EM_Disable;
-    } else if (!(brakePressure > MIN_BRAKE_PRESSURE)) {
-        DEBUG_PRINT("Failed to em enable, brake pressure low (%f)\n", brakePressure);
-        sendDTC_WARNING_EM_ENABLE_FAILED(1);
-        state = STATE_EM_Disable;
+    // } else if (!(brakePressure > MIN_BRAKE_PRESSURE)) {
```
Comment (jlim25) [inline vcu/Src/drive_by_wire.c:129]: This is commented out because of the brake fluid issue at comp right? Did we resolve that issue?

**Review by jlim25 — COMMENTED** [review]
Left some feedback and questions

File: pdu/Src/controlStateMachine_mock.c [hunk 2]
```diff
@@ -450,6 +372,40 @@ static const CLI_Command_Definition_t controlPumpsCommandDefinition =
     1 /* Number of parameters */
 };
 
+BaseType_t fakeHeatInv(char *writeBuffer, size_t writeBufferLength,
+                       const char *commandString)
+{
+    INV_Module_A_Temp = 45.0f;
+    INV_Module_B_Temp = 45.0f;
+    INV_Module_C_Temp = 45.0f;
+    fsmSendEvent(&mainFsmHandle, EV_EM_Enable, portMAX_DELAY);
```
Comment (addison-ch) [inline pdu/Src/controlStateMachine_mock.c:381]: If you transition to EM Enable right after faking the temperature, thats just going to turn the MC on and overwrite that with the real temperature readings isn't it?

File: vcu/Src/drive_by_wire.c [hunk 3]
```diff
@@ -242,6 +241,14 @@ uint32_t EM_Fault(uint32_t event)
...
             }
             break;
+        case EV_Inverter_Fault:
+            {
+                uint64_t faults = getInverterFaultCode();
+                sendDTC_CRITICAL_VCU_Inverter_Fault(faults);
+                // Check RMS GUI or CAN message 'MC_Fault_Codes' for fault details
+                DEBUG_PRINT("Inverter fault: %llu, trans to fatal failure\n", faults);
+                newState = STATE_Failure_Fatal;
+            }
```
Comment (addison-ch) [inline vcu/Src/drive_by_wire.c:251]: missing break :(

i thought this was fixed?

**Review by addison-ch — CHANGES_REQUESTED** [review]

**Review by aryanGajelli — APPROVED** [review]
This is a lot of changes, I'll give it a stamp for now as nothing looks broken to me from a quick glance

**Review by DanBishara — APPROVED** [review]

**Pull request merged**